source/documentation/continuous-integration-and-delivery/services.md

```diff
@@ -0,0 +1,14 @@
+## Services
+
+For convenience, the Reliability Engineering team provision the following
+resources for each Concourse team:
+
+- A private AWS S3 bucket
+- A public AWS S3 bucket
+- A private AWS Elastic Container Registry (ECR)
+
+Which can be accessed only by the individual Concourse team's worker instances.
```

Comment:
I assume this means the public one can be accessed by everyone from any IP and the private stuff can only be accessed from the concourse instances?

Reply:
Done in  1b2c382